simba/utils/enums.py

```diff
@@ -423,13 +391,13 @@ class Methods(Enum):
     THIRD_PARTY_ANNOTATION_FILE_NOT_FOUND = "Annotations data file NOT FOUND"
 
 
-class MetaKeys(Enum):
```

Comment:
In the meta files, the header is named "classifier_name", ideally I'd like to keep that to not break compatibility with config files that are created with older versions

Reply:
Correction, I change the value to align in both meta configs and the project config...